.yarnrc.yml

```diff
@@ -3,5 +3,9 @@ nodeLinker: node-modules
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
     spec: "@yarnpkg/plugin-workspace-tools"
+  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
+    spec: "@yarnpkg/plugin-interactive-tools"
+  - path: .yarn/plugins/@yarnpkg/plugin-outdated.cjs
+    spec: "https://mskelton.dev/yarn-outdated/v3"
```

Comment:
I have added this in order to be able to run [`yarn outdated`](https://github.com/mskelton/yarn-plugin-outdated#yarn-plugin-outdated) (which used to be available in yarn 1, but has been replaced with `yarn upgrade-interactive`, but without feature parity)